docs/local-development/debug-config.md

```diff
@@ -0,0 +1,101 @@
+# Debug
+
+## Difficulties
+It is difficult to debug/step-through front-end code so the recommendation is to use browser development tools separately. When using the IDE for debugging and setting breakpoints it should only be for back-end/server code.
+
+## Gotchas with IDE debugging
+The module aliasing (e.g. import * from "shared/...") is what throws off most debuggers and build tools. Usually `tsconfig` files are used by debuggers and you may need to create a separate tsconfig for your local environment so that the IDE knows how to run your app.
```

Comment:
May be worth adding a note how the root tsconfig file contains all of the aliases relative from the root of the repo. Also, the front-end is first compiled from TS to JS, then bundled with browserify. The bundling step uses the "pathmodify" plugin to enact the alias changes.

Reply:
agreed...do you want to add that note, while you're at it?